server.go

```diff
@@ -252,3 +257,122 @@ func successfulLogin(conf config.Config, delphisBackend backend.DelphisBackend)
 	}
 	return http.HandlerFunc(fn)
 }
+
+func getImage(delphisBackend backend.DelphisBackend) http.Handler {
+	fn := func(w http.ResponseWriter, r *http.Request) {
+		if r.Method != "POST" {
```

Comment:
need to pass in mimetype. do we want this as a URL parameter instead?

Reply:
This is confusing -- getImage requires a post? I would assume getImage was a get no?

Reply:
w.r.t. uploading image we should pass it in the `Content-Type` header, I think

Reply:
we can also check the filebytes to see what type of file it is or we could offload this to the client